### PR TITLE
Add gif logo

### PR DESCRIPTION
**Pull request by ayu-ayushisood**

### CODE REVIEW

File: src/components/Sidebar/style.css [hunk 1]
```diff
@@ -18,6 +21,11 @@
     text-decoration: none;
 }
 
+.logo {
+    height: 250px;
```
Comment (karanasthana) [inline src/components/Sidebar/style.css]: Can we use ‘em’ instead?

Comment (ayu-ayushisood) [reply src/components/Sidebar/style.css]: yes. done

**Review by karanasthana — APPROVED** [review]

File: src/components/Sidebar/index.js [hunk 2]
```diff
@@ -42,26 +38,19 @@ const Sidebar = () => {
...
-                />
-                <LogoInstagram
-                    height="25px"
-                    width="25px"
-                />
-                <LogoBehance
-                    height="25px"
-                    width="25px"
-                />
+                <a href="https://github.com/ayu-ayushisood" target="_blank">
```
Comment (karanasthana) [inline src/components/Sidebar/index.js]: Can we move these links a constants file ?

File: src/components/Sidebar/index.js [hunk 3]
```diff
@@ -42,26 +38,19 @@ const Sidebar = () => {
...
+                    <LogoGithub className="icons-style" />
+                </a>
+                <a href="https://www.linkedin.com/in/ayushi-sood" target="_blank">
```
Comment (karanasthana) [inline src/components/Sidebar/index.js]: Also, for all <a> tags, we should use `rel=noreferrer` for safety purposes

**Review by karanasthana — CHANGES_REQUESTED** [review]

File: src/components/Sidebar/index.js (hunk 3 above)
Comment (ayu-ayushisood) [reply src/components/Sidebar/index.js]: okay. done

File: src/components/Sidebar/index.js (hunk 2 above)
Comment (ayu-ayushisood) [reply src/components/Sidebar/index.js]: yes. done

**Review by karanasthana — APPROVED** [review]

**Pull request merged**